<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QRDC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrdc.qrc">
    <normaloff>:/icons/icons/qrdc.png</normaloff>:/icons/icons/qrdc.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>346</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Action</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDel"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionConnections"/>
    <addaction name="actionHosts"/>
    <addaction name="actionVariables"/>
    <addaction name="actionProtocols"/>
   </widget>
   <widget class="QMenu" name="menu_Service">
    <property name="title">
     <string>&amp;Service</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionBackupDB"/>
    <addaction name="actionRestoreDB"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Actions"/>
   <addaction name="menu_Service"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Tool bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDel"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionBackupDB"/>
   <addaction name="actionRestoreDB"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../qrdc.qrc">
     <normaloff>:/icons/icons/qt4.gif</normaloff>:/icons/icons/qt4.gif</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="toolTip">
    <string>Add new connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../qrdc.qrc">
     <normaloff>:/icons/icons/edit.gif</normaloff>:/icons/icons/edit.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit connection</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Return</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>&amp;Del</string>
   </property>
   <property name="toolTip">
    <string>Delete connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../qrdc.qrc">
     <normaloff>:/icons/icons/connect.gif</normaloff>:/icons/icons/connect.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open current connection</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionVariables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Variables</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+V</string>
   </property>
  </action>
  <action name="actionProtocols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Protocols</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionHosts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hosts</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+H</string>
   </property>
  </action>
  <action name="actionConnections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Connections</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+C</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../qrdc.qrc">
     <normaloff>:/icons/icons/config.gif</normaloff>:/icons/icons/config.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionBackupDB">
   <property name="icon">
    <iconset resource="../qrdc.qrc">
     <normaloff>:/icons/icons/backup.png</normaloff>:/icons/icons/backup.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Backup</string>
   </property>
   <property name="toolTip">
    <string>Backup database to text file</string>
   </property>
  </action>
  <action name="actionRestoreDB">
   <property name="icon">
    <iconset resource="../qrdc.qrc">
     <normaloff>:/icons/icons/restore.png</normaloff>:/icons/icons/restore.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore database from text file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrdc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>actionOpen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
